custom_components/tesla_custom/climate.py

```diff
@@ -124,3 +126,38 @@ def preset_modes(self) -> list[str] | None:
         Requires SUPPORT_PRESET_MODE.
         """
         return self.tesla_device.preset_modes
+
+    async def update_climate_related_devices(self):
+        """Reset the Manual Update time on climate related devices.
+
+        This way, their states are correctly reflected if they are dependant on the Climate state.
+        """
+
+        # This is really gross, and i kinda hate it.
+        # but its the only way i could figure out how to force an update on the underlying device
+        # thats in the teslajsonpy library.
+        # This could be fixed by doing a pr in the underlying library,
```

Comment:
By adding a public method to allow resetting the manual update time? 
Perhaps it's worth filing an issue on the `teslajsonpy` repo, so this doesn't get forgotten.

Reply:
Yes, let's open an issue. Basically what you're saying we need is a way to force a device to reread its info from cache.

I'm ok with the current approach for a 1.0 type release, but it makes sense to resolve it more cleanly in the future.